src/cmd/ksh93/RELEASE93

```diff
@@ -61,11 +61,11 @@ of ksh.
 	    performed arithmetic evaluation when the discipline
 	    was called from the arithmetic evaluator has been fixed.
 	e.  A bug caused by an EXIT trap inside a function that
-	    was executed in a subshell was fixed.
+	    was executed in a sub-shell was fixed.
```

Comment:
Actually, `subshell` is the more common spelling. And in as much as both are acceptable variants this is a gratuitous change.